README.md

```diff
@@ -158,6 +158,45 @@ and this to your crate root:
 extern crate hdrsample;
 ```
 
+## Benchmarking
+
+Both Rust's built-in benchmarking and [Criterion](https://github.com/japaric/criterion.rs) are used.
+
+For the benchmarks in `benches`:
+
+```
+rustup run nightly cargo bench
```

Comment:
I'd be happy to move all the benchmarks to just regular `test` benchmarks. I completely agree with you that the lack of development on Criterion isn't great.